modules/core/ICamera.h

```diff
@@ -3,6 +3,8 @@
 #pragma once
```

Comment:
This is an interface class. I think that at the current stage it should not be modified. Let us store the values (lens radius, focal distance and number of blades) in the derived camera class